sdk/go/common/resource/plugin/provider_server.go

```diff
@@ -360,7 +361,23 @@ func (p *providerServer) Configure(ctx context.Context,
 		}
 	}
 
-	if _, err := p.provider.Configure(ctx, ConfigureRequest{inputs}); err != nil {
+	var urn *resource.URN
+	if req.Urn != nil {
+		urnVal := resource.URN(*req.Urn)
+		urn = &urnVal
+	}
+	var id *resource.ID
+	if req.Id != nil {
+		idVal := resource.ID(*req.Id)
+		id = &idVal
+	}
+
+	_, err := p.provider.Configure(ctx, ConfigureRequest{
+		URN:    urn,
+		ID:     id,
+		Inputs: inputs,
```

Comment:
No type or name -- this is deliberate?

Reply:
No I missed it

Reply:
This is why we have others sanity check our work 😆